201-logic-app-xslt-with-params/azuredeploy.json

```diff
@@ -0,0 +1,188 @@
+      "type": "Microsoft.Logic/integrationAccounts/maps",
+      "apiVersion": "2015-08-01-preview",
+      "dependsOn": [
+        "[parameters('integrationAccountName')]"
+      ]
+    },
+    {
+      "name": "[parameters('logicAppName')]",
+      "type": "Microsoft.Logic/workflows",
+      "location": "[parameters('logicAppLocation')]",
+      "tags": {
+        "displayName": "LogicApp"
+      },
+      "apiVersion": "2016-06-01",
+      "properties": {
+        "state": "Enabled",
+        "integrationAccount": {
+          "id": "[resourceId('Microsoft.Logic/integrationAccounts', parameters('integrationAccountName'))]"
+        },
+        "definition": {
+          "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+          "actions": {
+            "Response": {
+              "inputs": {
+                "body": "@body('Transform_XML')",
+                "statusCode": 200
+              },
+              "runAfter": {
+                "Transform_XML": [
+                  "Succeeded"
+                ]
+              },
+              "type": "Response"
+            },
+            "Transform_XML": {
+              "inputs": {
+                "content": "@{triggerBody()}",
+                "function": {
+                  "id": "[concat(resourceId('microsoft.web/sites/', parameters('existingFunctionAppName')), '/functions/', parameters('transformFunctionName'))]"
+                },
+                "integrationAccount": {
+                  "map": {
+                    "name": "map-with-params"
+                  }
+                },
+                "xsltParameters": {
+                  "input:X": "20",
+                  "input:Y": "10"
+                }
+              },
+              "runAfter": { },
+              "type": "Xslt"
+            }
+          },
+          "contentVersion": "1.0.0.0",
+          "outputs": { },
+          "parameters": { },
+          "triggers": {
+            "manual": {
+              "inputs": {
+                "schema": { }
+              },
+              "kind": "Http",
+              "type": "Request"
+            }
+          }
+        }
+      },
+      "dependsOn": [
+        "[parameters('integrationAccountName')]"
+      ]
+    },
+    {
+      "apiVersion": "2015-08-01",
+      "name": "[concat(parameters('existingFunctionAppName'),'/', parameters('transformFunctionName'))]",
+      "type": "Microsoft.Web/sites/functions",
+      "properties": {
+        "config": {
+          "bindings": [
+            {
+              "name": "req",
+              "webHookType": "genericJson",
+              "direction": "in",
+              "type": "httpTrigger"
+            },
+            {
+              "name": "res",
+              "direction": "out",
+              "type": "http"
+            }
+          ]
+        },
+        "files": {
```

Comment:
This isn't possible today. I will fix it once we support alternate mechanisms in the RP (hopefully later this year).